packager/packager.cc

```diff
@@ -603,7 +605,12 @@ unique_ptr<PackageInfo> getPackageInfo(core::MutableContext ctx, ast::ParsedFile
     package.tree = ast::TreeMap::apply(ctx, finder, move(package.tree));
     finder.finalize(ctx);
     if (finder.info) {
-        finder.info->packagePathPrefix = packageFilePath.substr(0, packageFilePath.find_last_of('/') + 1);
+        finder.info->packagePathPrefixes.emplace_back(packageFilePath.substr(0, packageFilePath.find_last_of('/') + 1));
+        for (string prefix : extraPackageFilesDirectoryPrefixes) {
```

Comment:
This can be `for (const string &prefix : extraPackageFilesDirectoryPrefixes) {` to avoid copies.